azure/notebooks/kubernetes/Dask_Kube_cuML_Exploration_Full.ipynb

```diff
@@ -0,0 +1,1362 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    taxi_parquet_path = ["gs://anaconda-public-data/nyc-taxi/nyc.parquet/*.parquet"]
</pre><p>Is there a reason we're switching to use gcsfs data here? Since, it's an Azure example and we make use of Opendatasets in the other notebook it might be more relevant to use the same data.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cloud-ml-examples/pull/118/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
What Azure opendatasets does is that it first downloads the data on the local machine then we send it to the workers. This will not work for the second notebook where the data sets are huge, because of limitations of tornado buffer size. The GCFS links provides parquet and csv endpoints so that the workers can read their part remotely directly. If we have an Azure endpoint for the same purpose, then I think we can change it. But I don't know of one yet.

Reply:
I see, can we add a note in the notebook specifying this? Just so the user knows why we are using different datasets.

Reply:
Will do that .